rust/signer/src/lib.rs

```diff
@@ -280,6 +280,21 @@ fn encode_to_qr(payload: &[u8], is_danger: bool) -> anyhow::Result<Vec<u8>, Stri
     qrcode_static::png_qr(payload, sensitivity).map_err(|e| format!("{}", e))
 }
 
+/// Get all networks registered within this device
+fn get_all_networks(dbname: &str) -> anyhow::Result<Vec<MSCNetworkInfo>, String> {
```

Comment:
We have very similar logic in rust/db_handling/src/interface_signer.rs:396
Shouldn't we reuse it?